warehouse/core/src/main/java/datawave/iterators/filter/ageoff/DataTypeAgeOffFilter.java

```
@@ -295,12 +295,14 @@ protected void init(FilterOptions options, final long scanStart, IteratorEnviron
                 
                 final String dataTypeHasScanTime = options.getOption(dataType + ".hasScanTime");
                 if (Boolean.parseBoolean(dataTypeHasScanTime)) {
-                    final String scanTime = iterEnv.getConfig().get("table.custom.timestamp.current." + dataType);
-                    try {
-                        dataTypeScanTimes.put(dataType, Long.parseLong(scanTime, 10));
-                    } catch (final NumberFormatException e) {
-                        throw new NumberFormatException(dataType + " marked as hasScanTime but corresponding table.custom.timestamp.current." + dataType
-                                        + " is invalid: " + scanTime);
+                    if (iterEnv != null) {
```

Should we throw an exception if iterEnv is null?  That's effectively what would have happened before this change.

---

Sure -- not sure what the message should be, but I'll throw an NPE so it will work the same.